flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -163,7 +162,7 @@
 
 	/**
 	 * Amount of memory to be allocated by the task manager's memory manager. If not
-	 * set, a relative fraction will be allocated, as defined by {@link #LEGACY_MANAGED_MEMORY_FRACTION}.
+	 * set, a relative fraction will be allocated, as defined by {@link #MANAGED_MEMORY_FRACTION}.
```

Comment:
do we not want to add `withDeprecatedKeys("taskmanager.memory.fraction")` (`LEGACY_MANAGED_MEMORY_FRACTION `) to `MANAGED_MEMORY_FRACTION`?

Reply:
I don't think so. The semantic has changed. I think making it a deprecated key may mislead the user to overlook the semantic change.